profiles/common/modules/custom/tmgmt_workbench/tmgmt_workbench.module

```diff
@@ -114,55 +113,62 @@ function tmgmt_workbench_form_tmgmt_entity_ui_translate_form_alter(&$form, $form
       $state = $node->workbench_moderation['current']->state;
       $translation_possible = in_array($state, $states, 'strict');
 
-      $languages = tmgmt_available_languages();
-      $options = &$form['languages']['#options'];
+      $options = $form['languages']['#options'];
+
+      $existing_jobs = tmgmt_job_item_load_latest(
+        'workbench_moderation',
+        $entity_type,
+        $node->nid,
+        $node->translations->original
```

Comment:
It is only true with entity translation, not with I18n. As, the platform supports entity translation only that should be ok ... but question, what happens if a site as a content type with "Content translation"? Do we need to block the process somewhere?